hcatalog/streaming/src/test/org/apache/hive/hcatalog/streaming/mutate/client/lock/TestLock.java

```diff
@@ -321,7 +325,12 @@ public void testHeartbeatFailsTxnAbortedException() throws Exception {
   @Test
   public void testHeartbeatContinuesTException() throws Exception {
     Throwable t = new TException();
-    doThrow(t).when(mockMetaStoreClient).heartbeat(0, LOCK_ID);
+//    Following stubbings are unnecessary (click to navigate to relevant line of code):
+//    1. -> at org.apache.hive.hcatalog.streaming.mutate.client.lock.TestLock.testHeartbeatContinuesTException(TestLock.java:332)
+//    Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.
+
+//    This stub is unnecessary.
+//    doThrow(t).when(mockMetaStoreClient).heartbeat(0, LOCK_ID);
```

Comment:
if this line is uncommented, does it throw a UnnecessaryStubbingException ?

Reply:
Yes, that's correct. If we uncomment it we get the above error in the terminal output

```
//    Following stubbings are unnecessary (click to navigate to relevant line of code):
//    1. -> at org.apache.hive.hcatalog.streaming.mutate.client.lock.TestLock.testHeartbeatContinuesTException(TestLock.java:332)
//    Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.
```

It's confusing because the comment looks like instructions but it's the actual error that we got. I just copied-pasted in all occurences where I had to comment out the unnecessary stubs.

Reply:
Yeh that is very confusing.  Let's replace all these confusing comments with something like this:
```
This line causes a mockito UnnecessaryStubbingException. 
```

Reply:
I fixed it.